binancial/strategies/weighted_random.py

```
@@ -0,0 +1,32 @@
+        
+        '''
+        Initialize the Weighted Random strategy.
+
+        buy_weight | int | weight for the 'buy' action
+        sell_weight | int | weight for the 'sell' action
+        hold_weight | int | weight for the 'hold' action
+        '''
+        
+        self.weights = [buy_weight, sell_weight, hold_weight]
+
+    def run(self, ticker, accountant, price_usdt, timestamp):
```

Ruff (0.8.2)

Missing return type annotation for public function `run`

(ANN201)